### DISCUSSION TITLE
[v3] @persist directive reloads iframes

### DISCUSSION THREAD

**Discussion by PhiloNL** [livewire/livewire#5847] [7. Bugs (archived)]
https://github.com/livewire/next/issues/102
Not sure if this is a bug or expected behavior, but if you put in an iframe, e.g., an embedded YouTube video, inside the persist directive, it will reload the iframe when navigating between pages. The data in the example below is persisted, however. 

```blade
    @persist('foo')
    <div x-data="{ count: 1 }">
        <iframe width="560" height="315" src="https://www.youtube.com/embed/mH7cgoX3K0g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
        <span x-text="count" dusk="count"></span>
        <button x-on:click="count++" dusk="increment">+</button>
    </div>
    @endpersist
```

**Comment by joshhanley** [comment]
@PhiloNL I'm guessing this one will be hard to write a test for? 😬

**Reply by PhiloNL** [reply]
I'll give it a try 🤞🏻😬

**Reply by joshhanley** [reply]
Goodluck 🤞😆

**Reply by PhiloNL** [reply]
I took a bit to figure out but found a way to test it 😆 https://github.com/livewire/livewire/pull/5895

**Reply by joshhanley** [reply]
@PhiloNL legend!
